src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -230,17 +227,17 @@ public void withRMarkdown(String progressCaption, String userAction,
       deps.add(Dependency.cranPackage("digest", "0.6"));
```

Comment:
I believe we also want the `png` package in here (for some of the reticulate features). @kevinushey Is that in a separate PR?

Reply:
We request this in `withReticulate()`: https://github.com/rstudio/rstudio/blob/06f21da11d908c10130b5ac8170a8743afdd7ae7/src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java#L368

Reply:
Okay, great!